specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2023-08-01-preview/mfe.json

```diff
@@ -13277,6 +13330,27 @@
       },
       "additionalProperties": false
     },
+    "CapacityReservation": {
+      "required": [
+        "capacityReservationGroupId"
+      ],
+      "type": "object",
+      "properties": {
+        "capacityReservationGroupId": {
+          "description": "[Required] Specifies a capacity reservation group ID to allocate capacity from.",
+          "minLength": 1,
+          "pattern": "[a-zA-Z0-9_]",
+          "type": "string"
+        },
+        "endpointReservedCapacity": {
+          "format": "int32",
+          "description": "Specifies a capacity amount to reserve for this endpoint within the parent capacity reservation group.",
```

Comment:
>a capacity amount [](http://example.com/codeflow?start=36&length=17)

Units?

Reply:
it will be a throughput unit that matches the unit used in serverlessEndpoint.sku.capacity  and capacityReservationGroup.properties.reservedCapacity